### DISCUSSION TITLE
GUI controls doesn't work on DAW

### DISCUSSION THREAD

**Discussion by MikyBass** [apohl79/audiogridder#610] [Help]
Hi mates, how you doing.

I've been testing AG today. I really do love the idea. Everything seems to work fine, except for a thing: I can't move no buttons on the GUI of the plugin inside my DAW. I've tested it with native Logic plugins and Acustica, and none of it works. Works fine if I use the "Standard editor", but not in the GUI. If I do it in the server, the GUI works. So, I can work with, but it's not a nice workflow...

Don't know if I have some config done wrong or it's a bug.

I'm in Logic Pro 10.4.8 and AudioGridder 1.2.0 Beta 3. DAW is an iMac with High Sierra 10.13.6 and server is a MacBook Pro with Catalina 10.15.7, if it helps for some solution.

Thanks in advance!

**Comment by CaBuffalo** [comment]
Hi, you are the first one reporting this.

I only experienced it on Softube plugins on windows server with Logic.
Doesn't it work for any of the plugins you have?

On Fri, Jun 11, 2021 at 11:23 AM MikyBass ***@***.***> wrote:

> Hi mates, how you doing.
>
> I've been testing AG today. I really do love the idea. Everything seems to
> work fine, except for a thing: I can't move no buttons on the GUI of the
> plugin inside my DAW. I've tested it with native Logic plugins and
> Acustica, and none of it works. Works fine if I use the "Standard editor",
> but not in the GUI. If I do it in the server, the GUI works. So, I can work
> with, but it's not a nice workflow...
>
> Don't know if I have some config done wrong or it's a bug.
>
> I'm in Logic Pro 10.4.8 and AudioGridder 1.2.0 Beta 3. DAW is an iMac with
> High Sierra 10.13.6 and server is a MacBook Pro with Catalina 10.15.7, if
> it helps for some solution.
>
> Thanks in advance!
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/610>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6CGZ2SVRXL5AZZW4L3TSJIAJANCNFSM46RHZOBA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by MikyBass** [reply]
I've only tried the native Logic plugins and the Acustica Audio, 'cause I got no other installed on the server Mac. And none of them works. They work fine if if use it directly on the DAW, of course.

**Reply by MikyBass** [reply]
Is so weird that it works on standar editor but not on GUI, isn't it?

**Comment by apohl79** [comment]
> Hi mates, how you doing.
> 
> 
> 
> I've been testing AG today. I really do love the idea. Everything seems to work fine, except for a thing: I can't move no buttons on the GUI of the plugin inside my DAW. I've tested it with native Logic plugins and Acustica, and none of it works. Works fine if I use the "Standard editor", but not in the GUI. If I do it in the server, the GUI works. So, I can work with, but it's not a nice workflow...
> 
> 
> 
> Don't know if I have some config done wrong or it's a bug.
> 
> 
> 
> I'm in Logic Pro 10.4.8 and AudioGridder 1.2.0 Beta 3. DAW is an iMac with High Sierra 10.13.6 and server is a MacBook Pro with Catalina 10.15.7, if it helps for some solution.
> 
> 
> 
> Thanks in advance!

AG server is lacking the accessibility permission. Please follow https://audiogridder.com/server/#install point (2) to fix this.

**Reply by MikyBass** [reply]
It works! I put the programa on the list of "Accesibility" when Mac asked me, but didn't 'check' it later. What a fool...

Thank you so much!!
